vllm/model_executor/models/internlm2.py

```
@@ -144,27 +144,33 @@ def __init__(
         )
 
     def split_qkv(self, qkv: torch.Tensor):
-        seq_len = qkv.shape[0]
+        # Unpack all dimensions except the last one
+        *batch_dims, last_dim = qkv.shape
```

last_dim seems that is not used, please replace with _

---

The "Trigger Jenkins Tests" failure can be ignored now